pkg/runner/testdata/workdir/push.yml

```diff
@@ -5,13 +5,11 @@ jobs:
   workdir:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: ls -alFt "${GITHUB_WORKSPACE}/workdir"
+    - run: mkdir -p "${GITHUB_WORKSPACE}/workdir"
     - run: '[[ "$(pwd)" == "${GITHUB_WORKSPACE}/workdir" ]]'
       working-directory: workdir
 
   noworkdir:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
       - run: '[[ "$(pwd)" == "${GITHUB_WORKSPACE}" ]]'
```

Comment:
Why?

Reply:
Per top notes, actions checkout isn't required to verify the workdir test result works, the point of this test as I understand it is to verify the workdir context changes when `working-directory` is specified, so the checkout was extraneous.